src/pages/Filing/FilingApp/FilingWarnings/index.tsx

```diff
@@ -196,7 +198,10 @@ function FilingWarnings(): JSX.Element {
           There was an issue saving your Submission verification. Please click
           the &quot;Save and continue&quot; button to try again. If this issue
           persists,
-          <Link href={sblHelpMail}>contact our support staff</Link>.
+          <Link className='border-b-[1px] border-dotted' href={sblHelpMail}>
+            contact our support staff
+          </Link>
+          .
```

Comment:
I think wrapping this in a <p> tag should remove the need to apply border styles.